src/templates/en/base.html

```diff
@@ -88,6 +88,12 @@
 {% block author %}Author{% endblock %}
 {% block authors %}Authors{% endblock %}
 
+{% block reactions %}Reactions{% endblock %}
+{% block likes %}Likes{% endblock %}
+{% block reposts %}Reposts{% endblock %}
+{% block replies %}Replies{% endblock %}
+{% block mentions %}Mentions{% endblock %}
+
```

Comment:
I think the linter is complaining because these blocks should also be added to the other lang dirs (in English for now until translated)